src/index.html

```diff
@@ -6,9 +6,47 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <!-- <link rel="stylesheet" href="./reset.css"> -->
     <link rel="stylesheet" href="./style.css">
   </head>
+  <header class="header">
+    <div class="nav__logo">
+      <a href="#home">
+        <img class="img" src="images/logo.png" alt="logo">
+      </a>
+    </div>
+    <nav class="nav">
+      <ul class="nav__list">
+        <a href="#Apple" class="nav__link is-active">
+          <li class="nav__item">Apple</li>
```

Comment:
ul can have only li as child
```suggestion
      <ul class="nav__list">
        <li class="nav__item">
          <a href="#Apple" class="nav__link is-active">Apple</>  
        </li>
        
         
```